content/en/docs/concepts/security/index.md

```diff
@@ -516,14 +516,11 @@ access control for your workloads in the mesh. This level of control provides
 the following benefits:
 
 - Workload-to-workload and end-user-to-workload authorization.
-- A Simple API: it includes a single
-  [`AuthorizationPolicy` CRD](/docs/reference/config/security/authorization-policy/),
+- A Simple API: it includes a single [`AuthorizationPolicy` CRD](/docs/reference/config/security/authorization-policy/),
   which is easy to use and maintain.
-- Flexible semantics: operators can define custom conditions on Istio
-  attributes, and use DENY and ALLOW actions.
-- High performance: Istio authorization is enforced natively on Envoy.
-- High compatibility: supports gRPC, HTTP, HTTPS and HTTP2 natively, as well
-  as any plain TCP protocols.
+- Flexible semantics: operators can define custom conditions on Istio attributes, and use CUSTOM, DENY and ALLOW actions.
+- High performance: Istio authorization of DENY and ALLOW action is enforced natively on Envoy.
```

Comment:
Technically correct but at the expense of it being an easy-to-read overview.  Is it OK in a concepts page to leave it as it was?

Reply:
Updated to make this shorter, I called out `ALLOW` and `DENY` since the `CUSTOM` is not enforced in local Envoy.